controllers/Category.js

```diff
@@ -0,0 +1,25 @@
+const User = require('../models/User');
```

Comment:
And import only packages that are required: `User, Package, Iternary` don't seem to be required here.

Reply:
You haven't removed, why did you resolve it? @TheMightyRaider